gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/analytics/ApiAnalyticsResource.java

```diff
@@ -91,4 +96,18 @@ public ApiAnalyticsAverageConnectionDurationResponse getAverageConnectionDuratio
             .map(ApiAnalyticsMapper.INSTANCE::map)
             .orElseThrow(() -> new NotFoundException("No connection duration found for api: " + apiId));
     }
+
+    @Path("/status-codes-by-entrypoint")
```

Comment:
How about adding openapi config ? (like any others analytics and mAPI V4 endpoint) 

https://github.com/gravitee-io/gravitee-api-management/blob/edf9d16be914f878415c689c5264c999c900a7f5/gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml